compiler/AST/expr.cpp

```diff
@@ -3786,7 +3786,8 @@ void CallExpr::prettyPrint(std::ostream *o) {
         argList.last()->prettyPrint(o);
         *o << ")";
       } else if (strcmp(expr->unresolved, "chpl_build_bounded_range") == 0 ||
-                 strcmp(expr->unresolved, "chpl_build_partially_bounded_range") == 0 ||
+                 strcmp(expr->unresolved, "chpl_build_low_bounded_range") == 0 ||
+                 strcmp(expr->unresolved, "chpl_build_high_bounded_range") == 0 ||
                  strcmp(expr->unresolved, "chpl_build_unbounded_range") == 0) {
         argList.first()->prettyPrint(o);
         *o << "..";
```

Comment:
Turns out this code is used by chpldoc to create the return signature of functions with a fully specified array so only the fully bound case will trigger. Cleaned up with https://github.com/chapel-lang/chapel/pull/3176/files#diff-73700b233050fbea2ce37d1582e0be93R3789